pkg/auth/authz_handler_test.go

```diff
@@ -586,55 +578,25 @@ func updateCluster(ctx context.Context, cli *client.AssistedInstall) error {
 }
 
 func deregisterCluster(ctx context.Context, cli *client.AssistedInstall) error {
-	_, err := cli.Installer.DeregisterCluster(
+	_, err := cli.Installer.V2DeregisterCluster(
 		ctx,
-		&installer.DeregisterClusterParams{
+		&installer.V2DeregisterClusterParams{
 			ClusterID: strfmt.UUID(uuid.New().String()),
 		})
 	return err
 }
 
-func generateClusterISO(ctx context.Context, cli *client.AssistedInstall) error {
-	_, err := cli.Installer.GenerateClusterISO(
-		ctx,
-		&installer.GenerateClusterISOParams{
-			ClusterID:         strfmt.UUID(uuid.New().String()),
-			ImageCreateParams: &models.ImageCreateParams{},
-		})
-	return err
-}
-
-func downloadClusterISO(ctx context.Context, cli *client.AssistedInstall) error {
-	file, err := ioutil.TempFile("/tmp", "test")
+func downloadClusterFiles(ctx context.Context, cli *client.AssistedInstall) error {
+	file, err := ioutil.TempFile(Directory, FileName)
 	if err != nil {
 		return err
 	}
-	_, err = cli.Installer.DownloadClusterISO(
-		ctx,
-		&installer.DownloadClusterISOParams{
-			ClusterID: strfmt.UUID(uuid.New().String()),
-		},
-		file)
-	return err
-}
 
-func downloadClusterISOHeaders(ctx context.Context, cli *client.AssistedInstall) error {
-	_, err := cli.Installer.DownloadClusterISOHeaders(
-		ctx,
-		&installer.DownloadClusterISOHeadersParams{
-			ClusterID: strfmt.UUID(uuid.New().String()),
-		})
-	return err
-}
+	defer os.Remove(fmt.Sprintf("{%s}/{%s}", Directory, FileName))
```

Comment:
Use `file.Name()` here. There's some randomness to the resulting name to ensure that running in parallel won't give the same name.

Reply:
Done